packages/core-web/src/styles/markbind.css

```diff
@@ -336,6 +337,7 @@ h4,
 h5,
 h6 {
     margin-bottom: 1rem;
+    scroll-margin-top: var(--navbar-height);
```

Comment:
You can reuse the `--sticky-header-height` variable for this, which will also make it dynamic when the navbar height changes :)